app/assets/js/components/Work/Fileset/ActionButtons/Access.jsx

```diff
@@ -29,11 +46,19 @@ function MediaButtons({ fileSet }) {
           Edit structure (vtt)
         </Button>
       )}
+      {isVideo(fileSet) && (
+        <Button onClick={handleDownloadVideo} disabled={downloadStarted}>
+          <span className="icon">
+            <IconDownload />
+          </span>
+          Download Video
```

Comment:
@adamjarling -This could also be an audio file rather than video...

(The backend ticket spelled it out better that it was functionality for AV access files. https://github.com/nulib/repodev_planning_and_docs/issues/3292 )

Reply:
Ah, gotcha.  Easy enough change.  Thanks.